src/inspection/InspectionUsers.tsx

```diff
@@ -52,6 +52,10 @@ const InspectionUsers: React.FC<PropTypes> = observer(({ inspection }) => {
     }
   }, [inspection, user, toggleSubscribed, refetchRelations])
 
+  const stopEventPropagation = useCallback((e: React.MouseEvent) => {
+    e.stopPropagation()
+  }, [])
+
```

Comment:
If necessary to stop propagation here, but I would rather have the ExpandableBox component catch and stop the propagation and not have to do that separately in all components which use the expandable box.

```suggestion
  const onRefetchRelations = useCallback((e: React.MouseEvent) => {
    e.stopPropagation()
    refetchRelations()
  }, [refetchRelations])

```

Reply:
No longer have to stop propagation here